packages/alfa-dom/test/get-tab-sequence.spec.tsx

```diff
@@ -0,0 +1,109 @@
+import { jsx } from "@siteimprove/alfa-jsx";
+import { test } from "@siteimprove/alfa-test";
+import { getTabSequence } from "../src/get-tab-sequence";
+
+test("Computes the tab sequence of three non-prioritized elements", t => {
+  const a1 = <a href="#">Foo</a>;
+  const a2 = <a href="#">Bar</a>;
+  const input = <input type="text" value="Hello" />;
+
+  t.deepEqual(
+    getTabSequence(
+      <div id="root">
+        {a1}
+        {a2}
+        {input}
+      </div>,
+      <div id="context" />
```

Comment:
The context is supposed to be the top-most node of a given document:

```jsx
const context = (
  <div>
    {div}
    {a}
    {input}
  </div>
);

getTabSequence(context, context);
```

In test cases, the context is often the node being operated on as is the case above.